android/README.md

```diff
@@ -36,9 +49,21 @@ To try out the Ping Android SDK please look at one of our samples:
   (https://docs.pingidentity.com/sdks/latest/sdks/tutorials/android/index.html)
 
 
-### Integrate with PingOne:
+### Integrate with flows - PingOne DaVinci:
+
+- Coming soon
+
+### OIDC Login:
+
+- Coming soon
+
+## Documentation
+
+Detailed [documentation](https://docs.pingidentity.com/sdks/latest/sdks/index.html) is provided, and includes topics such as:
 
-- Samples coming soon
+- Introducing the SDK features
+- Preparing your server for use with the SDKs
```

Comment:
In the reorg I'm inlining most of the server setup with each tutorial, so it's not quite so obvious now.
Perhaps switch this to two bullets, such as:
 - Tutorial walkthroughs
 - Integrating functionality such as PingOne Protect, WebAuthn, and more.